test/TestProjects/FirstTest-TypeSpec/src/Generated/FirstTestTypeSpecClient.cs

```diff
@@ -534,18 +534,17 @@ public virtual Response FriendlyModel(RequestContent content, RequestContext con
         /// </item>
         /// </list>
         /// </summary>
-        /// <param name="repeatabilityFirstSent"> The DateTime to use. </param>
         /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
         /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
         /// <returns> The response returned from the service. </returns>
-        /// <include file="Docs/FirstTestTypeSpecClient.xml" path="doc/members/member[@name='AddTimeHeaderAsync(DateTimeOffset?,RequestContext)']/*" />
-        public virtual async Task<Response> AddTimeHeaderAsync(DateTimeOffset? repeatabilityFirstSent = null, RequestContext context = null)
+        /// <include file="Docs/FirstTestTypeSpecClient.xml" path="doc/members/member[@name='AddTimeHeaderAsync(RequestContext)']/*" />
+        public virtual async Task<Response> AddTimeHeaderAsync(RequestContext context = null)
```

Comment:
One thing to note that if only one of the headers is defined, we'll also remove it from the method parameters. Reasons are:
- Those are headers, not request content or query parameters. Service should not define their own headers with the same name.
- In some cases, defining only one header makes sense. For example, if the service doesn't care about the time, then only providing `Repeatability-Request-ID` is OK.
- Here are just test projects, not real ones.

From what I know, Java generator adopts the same strategy.